### DISCUSSION TITLE
Can Airflow SFTP Operator ingest data from Windows Server 2019 ?

### DISCUSSION THREAD

**Discussion by ashishmgofficial** [apache/airflow#29141] [Q&A]
We have a requirement where we have our source data in an external Windows Server 2019. Now I think that Win Server 2019 has support for SFTP via OpenSSH if im not wrong. Im wondering If I can use Airflow to ingest data using SFTP operator from this server to my S3 bucket .
Any thought would be helpful

**Comment by potiuk** [comment]
Maybe try this: https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/transfer/sftp_to_s3.html

**Comment by potiuk** [comment]
I think you won't know until you tried.
